client/components/convert-to-blocks/index.jsx

```diff
@@ -0,0 +1,71 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Dialog from 'components/dialog';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class ConvertToBlocksDialog extends Component {
+	static propTypes = {
+		close: PropTypes.func,
+		showDialog: PropTypes.bool,
+		confirmConversion: PropTypes.func,
+	};
+
+	shouldComponentUpdate( nextProps ) {
+		return this.props.showDialog !== nextProps.showDialog;
+	}
+
+	render() {
+		if ( this.props.isDirty ) {
```

Comment:
I don't see this property is defined/set ever, so we can likely remove this check.

Reply:
🤔 curious where this copy pasta came from